### PR TITLE
Showing Sub-organizations as dropdown from the organization of a Grower

### PR DESCRIPTION
**Pull request by samwel141**
## Description
managed to show the dropdown list of sub-organizations associated with an organization of a Grower

**Issue(s) addressed**
Issue number 886 under progress to be resolved

**What kind of change(s) does this PR introduce?**

- [x ] Enhancement
- [ ] Bug fix
- [x] Refactor

**Please check if the PR fulfils these requirements**

- [x ] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## Issue

**What is the current behavior?**

**What is the new behavior?**

now on checking an inline radio button the list of sub-organizations drops from organization name

## Breaking change

May close issue: [#886](https://github.com/Greenstand/treetracker-admin-client/issues/886

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -54,6 +54,7 @@
     "stream": "npm:stream-browserify",
     "typeface-roboto": "*",
     "uuid": "^9.0.1",
+    "webpack": "^5.89.0",
```
Comment (gwynndp) [inline package.json:57]: Was this intended?  It shouldn't be necessary, but if you ran into a problem then we can create a separate issue for it.

**Comment by gwynndp** [comment]
@samwel141 

Off to a nice start!  Can you link to the issue that this PR resolves?  
I think there are some open questions to answer before we can fully resolve this.

- Do we want to show all the Sub-Orgs for the Organization on each grower, or only show the Organization and Sub-Orgs that they belong to
- For a better UX we may need a way to tell the difference between the Org/Sub-Orgs the grower belongs to and the rest, if we want to list them all there

In the meantime, there is a display problem with your current implementation. It is also displaying on the Grower Cards in a jumbled way.  See below:

Screenshot 2023-11-24 at 12 45 11 PM

On the card we might just want to list all of the organizations as comma-separated values to keep it compact, rather than have a dynamic dropdown.

**Review by gwynndp — CHANGES_REQUESTED** [review]

**Comment by samwel141** [comment]
Hello, thank you so much for your consideration.
I was trying to solve this issue
https://github.com/Greenstand/treetracker-admin-client/issues/886 and my
concern is to know from you if that is what you wanted.
About the change fracturing the UI in cards, I'll work on it but it depends
on your reply to the above question.
Any change in the package.json file was not intended and will rectify it.

Regards,

**Comment by samwel141** [comment]
I have put everything clear in the UI can you give feedback on that also

**Comment by gwynndp** [comment]
Looks much better.  If you remove your package.json and package-lock.json changes then I'll merge.

**Comment by samwel141** [comment]
Removed changes in the package.json and package-lock.json, so it is now clear.

**Review by gwynndp — APPROVED** [review]

**Pull request merged**